crates/matrix-sdk-crypto/src/gossiping/machine.rs

```diff
@@ -1653,11 +1676,13 @@ mod tests {
             .unwrap()
             .get(&DeviceIdOrAllDevices::DeviceId(alice_device_id().to_owned()))
             .unwrap();
-        let content: ToDeviceRoomEncryptedEventContent = content.deserialize_as().unwrap();
+        let content: ToDeviceEncryptedEventContent = content.deserialize_as().unwrap();
 
         bob_machine.mark_outgoing_request_as_sent(id).await.unwrap();
 
-        let event = ToDeviceEvent { sender: bob_id().to_owned(), content };
+        let event =
+            ToDeviceEvent { sender: bob_id().to_owned(), content, other: Default::default() };
+        let event = Raw::new(&event).unwrap().deserialize_as().unwrap();
```

Comment:
Just to be clear: There's more instances of this pattern that could use the util function I mentioned, not just this one. Should I leave review comments on all of them?

Reply:
If you know where they are, sure.

Reply:
I would have to grep for them 🤷🏼

Reply:
In that case I'll try to find them myself.

Reply:
Hmm, this particular pattern with a ToDeviceEvent only appears twice, did see another pattern that could use the method?